src/azure-cli/azure/cli/command_modules/iot/tests/latest/test_iot_dps_commands.py

```diff
@@ -188,11 +193,19 @@ def test_dps_certificate_lifecycle(self, group_name, group_location):
                             self.check('properties.isVerified', True)
         ]).get_output_in_json()['etag']
 
+        # Create certificate from a chain - test how certificate is encoded in the service call
```

Comment:
Is there a part of the test that is supposed to assert certificate encoding like this comment suggests?